### PR TITLE
inspectrum: 20160403 -> 20170218

### PR DESCRIPTION
**Pull request by mbrgm**
And as a dependency:
liquid-dsp: init at 20170307

### CODE REVIEW

File: pkgs/development/libraries/liquid-dsp/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "liquid-dsp-${version}";
+  version = "20170307";
+
+  src = fetchFromGitHub {
+    owner = "jgaeddert";
+    repo = "liquid-dsp";
+    rev = "8c1978fa4f5662b8849fe712be716958f29cec0e";
```
Comment (Mic92) [inline pkgs/development/libraries/liquid-dsp/default.nix]: v1.3.0 is too old?

File: pkgs/applications/misc/inspectrum/default.nix [hunk 2]
```diff
@@ -1,18 +1,35 @@
...
+, fftwFloat
+, qt5
+, gnuradio
+, liquid-dsp
+}:
 
 stdenv.mkDerivation rec {
   name = "inspectrum-${version}";
-  version = "20160403";
+  version = "20170218";
```
Comment (Mic92) [inline pkgs/applications/misc/inspectrum/default.nix]: How about version 0.2, which was released on 3. February?

File: pkgs/development/libraries/liquid-dsp/default.nix (hunk 1 above)
Comment (mbrgm) [inline pkgs/development/libraries/liquid-dsp/default.nix]: v1.3.0 fails to compile:

```
/nix/store/ndqahbwrnabiaszk0s8cblhgwhid5kn0-gcc-5.4.0/lib/gcc/x86_64-unknown-linux-gnu/5.4.0/include/pmmintrin.h:50:1: error: inlining failed in call to always_inline '_mm_addsub_ps': target specific option mismatch
 _mm_addsub_ps (__m128 __X, __m128 __Y)
 ^
src/dotprod/src/dotprod_cccf.mmx.c:255:11: error: called from here
         s = _mm_addsub_ps( ci, cq );
           ^
In file included from src/dotprod/src/dotprod_cccf.mmx.c:49:0:
```

This seems to be fixed in the version I referenced.

File: pkgs/applications/misc/inspectrum/default.nix (hunk 2 above)
Comment (mbrgm) [inline pkgs/applications/misc/inspectrum/default.nix]: Version 0.2 has some bugs, which are fixed in latest git. So the question is if it's preferred to have the version with the tag or the one containing the bugfix...

**Comment by ndowens** [comment]
I would vote bugfix one but it's up to someone that can merge this. Just voting 😀

**Pull request merged**